creator-node/src/services/stateMachineManager/stateReconciliation/updateReplicaSet.jobProcessor.ts

```diff
@@ -87,26 +93,50 @@ const updateReplicaSetJobProcessor = async function ({
         enableIdentity: true,
         logger
       })
+    } catch (e) {
+      error = error || 'failure_init_audius_libs'
+      errorMsg = 'failed to init libs'
+      logger.error(`ERROR ${errorMsg} - ${(e as Error).message}`)
+    }
```

Comment:
nit prob more useful to put more info in `errorMsg` e.g.
```
errorMsg = `Failed to init audius libs - ${e.message}`
logger.error(errorMsg)
```
also matches pattern below on L117

Reply:
also if libsinit fails, then there's no point proceeding to the next try-catch block right? it's guaranteed to fail